### ISSUE TITLE
Include file field in submission table triggers error

### ISSUE THREAD

**Issue by rauwebieten** [solspace/craft3-freeform#1574]
### What happened?

In the submission overview >choose form > customize sources > try to include a file-drag-and-drop field.
Triggers error in the control panel

### Errors and Stack Trace (if available)

```shell
2024-10-16 09:28:48 [web.ERROR] [Twig\Error\RuntimeError] Twig\Error\RuntimeError: Calling unknown method: craft\elements\db\AssetQuery::thumbUrl() in /var/www/vendor/solspace/craft-freeform/packages/plugin/src/templates/_components/fields/file.html:3
Stack trace:
#0 /var/www/storage/runtime/compiled_templates/53/53c6d924e319171af606806f8701ed63.php(50): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(craft\elements\db\AssetQuery), 'thumbUrl', Array, 'method', false, false, false, 3)
#1 /var/www/vendor/twig/twig/src/Template.php(393): __TwigTemplate_c0291e057ef128be035758194125887b->doDisplay(Array, Array)
#2 /var/www/vendor/twig/twig/src/Template.php(349): Twig\Template->yield(Array, Array)
#3 /var/www/vendor/twig/twig/src/Template.php(364): Twig\Template->display(Array)
#4 /var/www/vendor/twig/twig/src/TemplateWrapper.php(35): Twig\Template->render(Array)
#5 /var/www/vendor/twig/twig/src/Environment.php(306): Twig\TemplateWrapper->render(Array)
#6 /var/www/vendor/craftcms/cms/src/web/View.php(501): Twig\Environment->render('freeform/_compo...', Array)
#7 /var/www/vendor/solspace/craft-freeform/packages/plugin/src/Bundles/Fields/Implementations/FileUpload/FileUploadBundle.php(98): craft\web\View->renderTemplate('freeform/_compo...', Array)
#8 [internal function]: Solspace\Freeform\Bundles\Fields\Implementations\FileUpload\FileUploadBundle->renderTableValue(Object(Solspace\Freeform\Events\Submissions\RenderTableValueEvent))
#9 /var/www/vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Array, Object(Solspace\Freeform\Events\Submissions\RenderTableValueEvent))
#10 /var/www/vendor/yiisoft/yii2/base/Component.php(650): yii\base\Event::trigger('Solspace\\Freefo...', 'render-submissi...', Object(Solspace\Freeform\Events\Submissions\RenderTableValueEvent))
#11 /var/www/vendor/solspace/craft-freeform/packages/plugin/src/Elements/Submission.php(877): yii\base\Component->trigger('render-submissi...', Object(Solspace\Freeform\Events\Submissions\RenderTableValueEvent))
#12 /var/www/vendor/craftcms/cms/src/base/Element.php(5237): Solspace\Freeform\Elements\Submission->attributeHtml('field:8')
#13 /var/www/vendor/twig/twig/src/Extension/CoreExtension.php(1754): craft\base\Element->getAttributeHtml('field:8')
#14 /var/www/vendor/craftcms/cms/src/helpers/Template.php(148): Twig\Extension\CoreExtension::getAttribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(Solspace\Freeform\Elements\Submission), 'getAttributeHtm...', Array, 'method', false, false, false, 106)
#15 /var/www/storage/runtime/compiled_templates/fb/fbc94619812035aad3a206ca57b83029.php(242): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(Solspace\Freeform\Elements\Submission), 'getAttributeHtm...', Array, 'method', false, false, false, 106)
#16 /var/www/vendor/twig/twig/src/Extension/CoreExtension.php(1919): __TwigTemplate_72a6862424f71a1e7746c66b72ac256e->{closure}()
#17 /var/www/storage/runtime/compiled_templates/fb/fbc94619812035aad3a206ca57b83029.php(43): Twig\Extension\CoreExtension::captureOutput(Object(Generator))
#18 /var/www/vendor/twig/twig/src/Template.php(393): __TwigTemplate_72a6862424f71a1e7746c66b72ac256e->doDisplay(Array, Array)
#19 /var/www/storage/runtime/compiled_templates/66/6670fa1524eff4f2c88aa0d640b7b925.php(159): Twig\Template->yield(Array)
#20 /var/www/vendor/twig/twig/src/Template.php(393): __TwigTemplate_000d3a7048bf312b2bd31f903d0c6705->doDisplay(Array, Array)
#21 /var/www/vendor/twig/twig/src/Template.php(349): Twig\Template->yield(Array, Array)
#22 /var/www/vendor/twig/twig/src/Template.php(364): Twig\Template->display(Array)
#23 /var/www/vendor/twig/twig/src/TemplateWrapper.php(35): Twig\Template->render(Array)
#24 /var/www/vendor/twig/twig/src/Environment.php(306): Twig\TemplateWrapper->render(Array)
#25 /var/www/vendor/craftcms/cms/src/web/View.php(501): Twig\Environment->render('_elements/table...', Array)
#26 /var/www/vendor/craftcms/cms/src/base/Element.php(1293): craft\web\View->renderTemplate('_elements/table...', Array)
#27 /var/www/vendor/craftcms/cms/src/controllers/ElementIndexesController.php(855): craft\base\Element::indexHtml(Object(Solspace\Freeform\Elements\Db\SubmissionQuery), Array, Array, 'form:2', 'index', true, true, false)
#28 /var/www/vendor/craftcms/cms/src/controllers/ElementIndexesController.php(176): craft\controllers\ElementIndexesController->elementResponseData(true, true)
#29 [internal function]: craft\controllers\ElementIndexesController->actionGetElements()
#30 /var/www/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#31 /var/www/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#32 /var/www/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('get-elements', Array)
#33 /var/www/vendor/craftcms/cms/src/web/Application.php(350): yii\base\Module->runAction('element-indexes...', Array)
#34 /var/www/vendor/craftcms/cms/src/web/Application.php(649): craft\web\Application->runAction('element-indexes...', Array)
#35 /var/www/vendor/craftcms/cms/src/web/Application.php(312): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#36 /var/www/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#37 /var/www/web/index.php(12): yii\base\Application->run()
```

### How can we reproduce this?

In the submission overview >choose form > customize sources > try to include a file-drag-and-drop field.

### Freeform Edition

Pro

### Freeform Version

Freeform PRO 5.6.3

### Craft Version

Craft Pro 5.4.6

### When did this issue start?

Unsure

**Comment by kjmartens** [comment]
Sorry about this @rauwebieten.

We'll get this resolved shortly. 🙂

**Comment by kjmartens** [comment]
This should now be resolved in Freeform 5.6.7. 🙂

**Comment by rauwebieten** [comment]
Thanks, works
